hledger/test/cli/commodity-style.test

```diff
@@ -0,0 +1,49 @@
+# Test whether only the style without a symbol is changed
+<
+2021-07-09 no symbol
+    (a)   123
+
+2021-07-09 Euro
+    (a)    EUR 1.234,56
+
+2021-07-09 Dollar
+    (a)      $ 1,234.56
+
+$ hledger -f- print -c '1 0'
+>
+2021-07-09 no symbol
+    (a)           1 2 3
```

Comment:
I think I might need some clarification with what's going on here. Here we're changing to digit groupings of size one with a space for a ‘thousands’ separator? Odd, but reasonable.

Reply:
To be clear, a space for a thousands separator is the only reasonable option ^^. I meant the digit groupings of sizse one was odd.

Reply:
Yeah, although an interesting corner case, this is probably not the place to test it.